docs/source-app/api_references.rst

```diff
@@ -32,10 +32,10 @@ ___________________
     :nosignatures:
     :template: classtemplate_no_index.rst
 
-    ~serve.serve.ModelInferenceAPI
     ~python.popen.PopenPythonScript
-    ~serve.gradio.ServeGradio
     ~python.tracer.TracerPythonScript
+    ~serve.gradio.ServeGradio
+    ~serve.serve.ModelInferenceAPI
```

Comment:
these were ordered a-Z.. Any reason why this is updated?

Reply:
Yes, let's have them order.